pkg/apis/core/v1alpha1/types_landscaperdeployment.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: 2021 "SAP SE or an SAP affiliate company and Gardener contributors"
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	lsschema "github.com/gardener/landscaper/apis/schema"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// The LandscaperDeployment is created to define a deployment of the Landscaper Service.
+type LandscaperDeployment struct {
```

Comment:
If this defines the deployment of a landscaper service and not only a landscaper, is LandscaperServiceDeployment perhaps a better name?

Reply:
It defines an instance of a Landscaper. The Landscaper Service Controller will pick this CRD up to install a Landscaper for a tenant.

Reply:
I also think landscaper is correct as the outcome is a Landscaper instance.
The service is the controller who reacts on a deployment as Rene mentioned

Reply:
then only the comment "a deployment of the Landscaper Service" was misleading for me

Reply:
@achimweigel Ah, I see. I will change the comment then.